client/src/models/imageScrubber.js

```diff
@@ -83,13 +96,16 @@ const imageScrubber = {
         this.receiveLocation(null, { id: payload.id, address: 'cached' })
       }
     },
-    async markTreeInactive(id) {
+    async toggleTreeActive(id, bool) {
```

Comment:
maybe instead of bool, `isActive`?

Reply:
i agree, let's use descriptive variable names.

Reply:
👍